### DISCUSSION TITLE
Search highlighting

### DISCUSSION THREAD

**Discussion by Yelinz** [marimo-team/marimo#679] [Ideas]
Currently it is hard to see where the matches from the search are.
Giving the option to highlight the matches would improve usability.

**Comment by mscolnick** [comment]
I will fix this - we currently only highlight one cell at a time, but we should highlight across all.

**Comment by mscolnick** [comment]
Search highlighting is improved now from https://github.com/marimo-team/marimo/releases/tag/0.1.88
